module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderButton.kt

```diff
@@ -22,6 +26,8 @@ import org.cru.godtools.shared.tool.parser.model.Dimension
 internal fun ColumnScope.RenderButton(button: Button, state: State) {
     val scope = rememberCoroutineScope()
 
+    val invisible by remember(button, state) { button.isInvisibleFlow(state) }.collectAsState(false)
+
```

Comment:
to avoid repeating this for every element, move this logic into the `invisibleIf` modifier. you will need to mark it as `@Composable` and should remove the `inline` keyword from it